protocol/contracts/lpp/src/state/config.rs

```diff
@@ -1,19 +1,18 @@
+use platform::contract::Code;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
 use currency::Currency;
 use finance::{percent::bound::BoundToHundredPercent, price::Price};
 use lpp_platform::NLpn;
-use sdk::{
-    cosmwasm_std::{Storage, Uint64},
-    cw_storage_plus::Item,
-};
+use sdk::{cosmwasm_std::Storage, cw_storage_plus::Item};
 
 use crate::{borrow::InterestRate, error::Result, msg::InstantiateMsg};
 
 #[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq)]
 pub struct Config {
     lpn_ticker: String,
-    lease_code_id: Uint64,
+    #[serde(alias = "lease_code_id")]
```

Comment:
Perhaps it'd be good to put a TODO here referencing the next release, after which the alias should be removed.